src/coreclr/gc/CMakeLists.txt

```diff
@@ -1,12 +1,16 @@
 set(CMAKE_INCLUDE_CURRENT_DIR ON)
 
-# Local GC meta-issue: https://github.com/dotnet/runtime/issues/8061
+if(FEATURE_STANDALONE_GC)
+  add_definitions(-DFEATURE_STANDALONE_GC)
 
-# https://github.com/dotnet/runtime/issues/8059
-remove_definitions(-DSTRESS_HEAP)
+  # Local GC meta-issue: https://github.com/dotnet/runtime/issues/8061
```

Comment:
we should probably take this opportunity and get rid of this out of date comment :)

Reply:
Just this one or all three of them? And should we also close the github issue 8061?

Reply:
all 3 of them please. and yes we can also close the issue. thanks!

Reply:
Removed and closed the issue.